lib/itamae/recipe.rb

```diff
@@ -80,6 +80,12 @@ def method_missing(*args, &block)
       end
 
       def define(name, params = {}, &block)
+        class_name = "Itamae::Resource::#{Resource.get_resource_class_name(name)}"
+        if Module.const_defined?(class_name) && Module.const_get(class_name).is_a?(Class)
```

Comment:
First, thank you for your PR.

I think the following is better: (I didn't check the code works or not)

```ruby
class_name = Resource.get_resource_class_name(name)
if Resource.const_defined?(class_name)
  ...
  return
end

Resource.const_set(class_name, Definition.create_class(name, params, &block))
```

What do you think?